frontend/src/views/main/project/InspectorLauncher.vue

```diff
@@ -16,6 +16,15 @@
         </v-radio-group>
       </div>
       <div v-else>No dataset uploaded yet on this project.</div>
+      <v-spacer/>
+      <div v-if="creatingInspection">
+        <span>Scoring the entire dataset. This operation can take a while...</span>
```

Comment:
Would just need to add "Scoring the entire dataset of {fileSizeInHumanReadableFormat}"